lib/turbo/engine.rb

```diff
@@ -92,8 +92,8 @@ class Engine < Rails::Engine
     end
 
     initializer "turbo.broadcastable" do
-      ActiveSupport.on_load(:active_job) do
-        ActiveSupport.on_load(:active_record) do
+      ActiveSupport.on_load(:active_record) do
+        if defined?(ActiveJob)
```

Comment:
There would still be a load order issue if someone loaded turbo-rails, then activated ActiveRecord and then loaded ActiveJob. The problem before was that ActiveJob would not get activated (and the ActiveSupport on_load) would not get triggered until *after* one of the broadcast macros was invoked. Because `require "active_job"` defines `ActiveJob` immediately, this works around it. I changed the order so that it wouldn't check until after ActiveRecord was activated. This means someone should be able to require turbo-rails prior to active job/active record and it'd still work assuming active record  isn't activated.

Reply:
Let me double check my assumption about active record's on_load

Reply:
Actually yeah, so in order for this to be a problem, they would actually have to not require active_job until after Rails was initialized. I'm pretty sure all sorts of things would go sideways if they did that, so I don't know that that's necessary to account for.

If you'd like to get this merged and discuss other ways to approach this, I'd be happy to.

Reply:
Okay, we'll get this out as a rush fix for now, and can then review.

Reply:
Sounds good, thanks. I just added you on LinkedIn, ping me there if you want to have a quick call about it, may be easier than GitHub back and forth.